Vtol improvements (#1378)
PX4/PX4-user_guide: en/frames_vtol/README.md

![Quantum Vector: Tiltroto VTOL](../../assets/airframes/vtol/quantum_vector/hero.png)

VTOL vehicles offer the benefits of both multicopter and fixed-wing flight:
- **Vertical takeoff and landing:** Even inexperienced pilots can take off and land virtually anywhere.
- **Fast and efficient fixed wing flight:** Faster, further, and longer missions, carrying heavier payloads.
- **Hovering:** Steady platform for photography, structure scans etc.

This section contains build logs and instructions for assembling and configuring a number of VTOL vehicle frames.

## VTOL Types

PX4 supports the three most important/main VTOL configurations.

<style>
.wrapper {
  display: grid;
  grid-template-columns: 1fr 1fr 1fr;
  grid-auto-rows: minmax(100px, auto);
}

.grid_item {
  border:1px solid gray;
  border-radius: 5px;
  background-color: #dfe2e5;
  margin: 5px;
  padding: 5px;
}

.grid_item_heading
{
  font-weight: bold;
  margin-right: auto;
  margin-left: auto;
  padding: 5px;
}

.wrapper img {
  margin-top: 10px;
}
</style>

> <div class="wrapper">

Author: Note this renders nicely in vuepress but not github. Sorry!

Author: It looks like this
![image](https://user-images.githubusercontent.com/5368500/126600739-ff14cbfe-6e2e-4e40-8eae-2b9eec902aff.png)

Reviewer: >It looks like this

Love it!

VTOL vehicles offer the benefits of both multicopter and fixed-wing flight:
- **Vertical takeoff and landing:** Even inexperienced pilots can take off and land virtually anywhere.
- **Fast and efficient fixed wing flight:** Faster, further, and longer missions, carrying heavier payloads.
- **Hovering:** Steady platform for photography, structure scans etc.

This section contains build logs and instructions for assembling and configuring a number of VTOL vehicle frames.

## VTOL Types

PX4 supports the three most important/main VTOL configurations.

<style>
.wrapper {
  display: grid;
  grid-template-columns: 1fr 1fr 1fr;
  grid-auto-rows: minmax(100px, auto);
}

.grid_item {
  border:1px solid gray;
  border-radius: 5px;
  background-color: #dfe2e5;
  margin: 5px;
  padding: 5px;
}

.grid_item_heading
{
  font-weight: bold;
  margin-right: auto;
  margin-left: auto;
  padding: 5px;
}

.wrapper img {
  margin-top: 10px;
}
</style>

<div class="wrapper">

  <div class="grid_item">
    <div class="grid_item_heading">Tailsitter</div>
    <div class="grid_text">
    Rotors permanently in fixed wing-position.

>     Takes off and lands on tail. Flips to fixed wing orientation for normal flight.

Reviewer: ```suggestion
    Takes off and lands on tail. Whole vehicle tilts forward to enter forward flight.

```

Author: Much better!

    <img src="../../assets/airframes/vtol/wingtraone/hero.jpg" title="wingtraone" />
    <ul>

>       <li>Simple and robust</li>

Reviewer: ```suggestion
      <li>Simple and robust, allows for compact design</li>

```

>       <li>Fewer actuators</li>

Reviewer: ```suggestion
      <li>Minimal set of actuators</li>

```

      <li>Can be hard to control, particularly in wind</li>

>       <li>Poor hovering</li>

Reviewer: ```suggestion
      <li>Tradeoff between efficiency in hover and forward flight, as same actuators are used</li>

```

    </ul>
    </div>
  </div>

<div class="grid_item">
  <div class="grid_item_heading">Tiltrotor</div>
  Rotors swivel 90 degrees to transition from multicopter to forward flight orientation.
  Takes off and lands on belly.
  <div class="grid_text">
  <img src="../../assets/airframes/vtol/eflite_convergence_pixfalcon/hero.jpg" title="Eflight Confvergence" />
  <ul>

>     <li>Mechanically more complex with additional actuators</li>

Reviewer: ```suggestion
    <li>Additional actuators for motor tilts</li>
    <li>Mechanically complex tilting mechanism</li>

```

>     <li>Easier to control than tailsitter</li>

Reviewer: ```suggestion
    <li>Easier to control in hover than tailsitters due to more control authority</li>

```

>     <li>Can hover</li>

Reviewer: ```suggestion

```

  </ul>
  </div>
</div>

<div class="grid_item">
  <div class="grid_item_heading">Standard VTOL</div>
  <div class="grid_text">
  Separate rotors/flight controls for multicopter and forward flight. Takes off and lands on belly.
  <img src="../../assets/airframes/vtol/vertical_technologies_deltaquad/hero_small.png" title="Vertical Technologies: Deltaquad" />
  <ul>

>     <li>Most complex. More things to go wrong but better redundancy.</li>

Reviewer: ```suggestion
    <li>Additional weight from separate hover/forward flight propulsion systems</li>

```

>     <li>Easiest to control</li>

Reviewer: ```suggestion
    <li>Easiest to control due to dedicated hover/forward flight actuators</li>

```

>     <li>Can hover</li>

Reviewer: ```suggestion
    <li>Add possibility for actuator redundancy</li>

```

>     <li>Fuel engines can be used</li>

Reviewer: ```suggestion
    <li>Fuel engines for forward flight propulsion can be used</li>

```

The VTOL codebase is the same codebase as for all other airframes and just adds additional control logic, in particular for transitions.

## Flying and Flight Modes

A VTOL aircraft can fly as either a multicopter or as fixed-wing vehicle.
Multicopter mode is mainly used for take off and landing while the fixed wing mode is used for efficient travel and/or mission execution.

The flight modes for VTOL vehicles are the same as for [multicopter](../getting_started/flight_modes.md#mc_flight_modes) when flying in MC mode and [fixed-wing](../getting_started/flight_modes.md#fw_flight_modes) when flying in FW mode.

The switch between modes is initiated either by the pilot using an RC switch or automatically by PX4 when needed in missions or other auto modes.

## Assembly

:::note
For information about commercial and kit VTOL vehicles see: [Complete Vehicles](../complete_vehicles/README.md) 
:::

PX4 controlled vehicles generally share the same core components: a flight controller connected to a power system, GPS, external compass (highly recommended), radio control system (optional) and/or telemetry radio system (optional), and airspeed sensor (highly recommended for VTOL vehicles).

> The flight controller outputs are connected to the vehicle motor ESCS and/or flight control servos and actuators, which are separately powered.

Reviewer: ```suggestion
The flight controller outputs are connected to the vehicle motor ESCs and/or flight control servos and actuators, which are separately powered.

```
